src/groupByKey.ts

```diff
@@ -2,6 +2,20 @@ interface Item {
   [key: string]: any;
 }
 
-export function groupByKey(items, field) {
-  // write code here;
+interface Group {
+  [key: string]: Item;
```

Comment:
```suggestion
  [key: string]: Item[];
```
Це ж згруповані айтеми, а не один